src/components/Add.js

```diff
@@ -0,0 +1,39 @@
+import React,{useState} from 'react';
+
+export default function Add(props) {
+    // Adding a hook to get the input and stores it.
+    const [input, setInput] = useState('');
+    
+    // Function that add items to the list
+    const addItem = () => {
```

Comment:
A common name convention when triggering a function from an `eventHandler` is to use `onAddItem`